src/network/networkd-link.c

```diff
@@ -2163,6 +2164,17 @@ static int link_set_can(Link *link) {
                         return log_link_error_errno(link, r, "Could not append IFLA_CAN_RESTART_MS attribute: %m");
         }
 
+        cm.mask = CAN_CTRLMODE_3_SAMPLES;
```

Comment:
why not move this up into the variable declaration? if it's set unconditionally, you might as well set it right away when you declare the variable